app/reviews/models.py

```diff
@@ -26,6 +26,9 @@ class Review(TimestampMixin):
 
     # Make nullable for anonymity of reviewers.
     user = models.ForeignKey(User, on_delete=models.SET_NULL, blank=True, null=True)
+    # anonymity_id is just a unique identifier for the reviewer used
+    # as a check prevent multiple reviews from one person
+    anonymity_id = models.CharField(blank=True, null=True, max_length=250)
```

Comment:
I think the purpose of preventing multiple reviews from a single user is defeated if this field is nullable or can be blank? Although the client can enforce that an anonymity_id be set, it wouldn't hurt if it's enforced at the  database level.